ansible/roles_ocp_workloads/ocp4_workload_cloud_native_robot/templates/cicd/applicationset.yaml.j2

```diff
@@ -0,0 +1,44 @@
+---
+apiVersion: argoproj.io/v1alpha1
+kind: ApplicationSet
+metadata:
+  name: cicd
+  namespace: openshift-gitops
+spec:
+  generators:
+  - list:
+      elements:
+{% for n in range(1, ocp4_workload_cloud_native_robot_shared_gitea_user_count | int + 1 ) %}
+      - user: {{ ocp4_workload_cloud_native_robot_shared_gitea_user_prefix }}{{ n }}
+{% endfor %}
+  template:
+    metadata:
+      name: "cicd-{% raw %}{{ user }}{% endraw %}"
+      namespace: openshift-gitops
+      finalizers:
+      - resources-finalizer.argocd.argoproj.io
+    spec:
+      destination:
+        name: ''
+        namespace: "{{ ocp4_workload_cloud_native_robot_shared_pipeline_namespace_prefix }}{% raw %}{{ user }}{% endraw %}"
+        server: 'https://kubernetes.default.svc'
+      project: default
+      syncPolicy:
+        automated:
+          prune: false
+          selfHeal: false
+      source:
+        repoURL: "http://{{ ocp4_workload_cloud_native_robot_shared_gitea_instance }}.{{ ocp4_workload_cloud_native_robot_shared_gitea_project }}.svc:3000/{% raw %}{{ user }}{% endraw %}/{{ ocp4_workload_cloud_native_robot_shared_gitea_repo }}"
+        targetRevision: main
```

Comment:
same here?

Reply:
Thanks, I removed the file. Because it's not in use. (copy&paste)...